crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -293,7 +348,17 @@ def upgrade_controller_nodes
         # and the existing cluster has been cleaned up by deleting most of resources
         return false unless node_api.post_upgrade
 
-        # FIXME: if upgrade went well, continue with next node(s)
+        # Delete router namespaces on non-upgraded node
+        master_node_api.delete_router_namespace
```

Comment:
So... do we need a different action for `pacemaker_transaction "neutron ha tool"` in case of `CrowbarPacemakerHelper.being_upgraded?`

https://github.com/crowbar/crowbar-openstack/blob/master/chef/cookbooks/neutron/recipes/network_agents_ha.rb#L203

If we are able to set the target role as stopped, we could be able to start it once we want...